distribution/docs/src/main/resources/_contents/_sources/federated-sources-contents.adoc

```diff
@@ -0,0 +1,11 @@
+
+=== Included Federated Sources
```

Comment:
Only the WFS v1.0.0 Source and WFS v2.0.0 Source links in this list actually link to their respective sections.